app/controllers/ops_controller/diagnostics.rb

```diff
@@ -350,6 +347,41 @@ def diagnostics_server_list
 
   private ############################
 
+  def fetch_local_log(*params)
+    if MiqEnvironment::Command.supports_systemd?
+      fetch_journal_log(*params)
+    elsif !MiqEnvironment::Command.is_podified?
+      fetch_log_file(*params)
+    end
```

Comment:
What happens if this falls through?

Reply:
It returns `nil` and you'll see `Logs for this %{product} Server are not available for viewing"` from https://github.com/ManageIQ/manageiq-ui-classic/pull/8538/files#diff-a3bbc06cb13f2323c82b7017b75a44f75f70db840472143881166a0001667393R680

Reply:
![image](https://user-images.githubusercontent.com/12851112/203411744-a664c3d3-5a1f-4827-af64-370907d369dc.png)